website/docs/find_a_job/hiring-process.mdx

```diff
@@ -0,0 +1,35 @@
+---
+id: hiring-process
+title: Le processus d'embauche
+slug: /le-processus-d-embauche
+---
+
+import Timeline from '@site/src/components/timeline.js';
+import TimelineStep from '@site/src/components/timeline_step.js';
+import Link from '@docusaurus/Link';
+
+Voici un petit descriptif du processus courant d'embauche, ça diffère d'une entreprise à l'autre, mais dans les grandes lignes ça se résume à ces quelques étapes.
```

Comment:
Pour ça diffère d'une entreprise à l'autre, tu peux mettre le lien vers ce site qui décrit plusieurs processus d'embauche: https://yangshun.github.io/tech-interview-handbook/interview-formats#formats-of-famous-companies

```html
<a href="https://yangshun.github.io/tech-interview-handbook/interview-formats#formats-of-famous-companies">ça diffère d'une entreprise à l'autre</a>
```
(je me suis rendu compte que Docusaurus mettait automatiquement les noreel noopener ^^")

Reply:
Ah oui je vais le mettre :+1: 
Il est bien ce petit Docusaurus ^^